2.-Configuration.md

```diff
@@ -50,6 +50,11 @@ We can disable the logging for all your services by using `-Dts.global.log.enabl
 
 By default logs are using ANSI colors, use `-Dts.global.log.nocolor=true` to disable them.
 
+# Service directory
+
+Some services crate their own folder under the `target` directory, which will contain temporary files (e.g. CLI or DEV mode services).
+If you want the service to delete the folder on exit, set `ts.global.delete.folder.on.exit` to `true`.
```

Comment:
```suggestion
If you want service to delete the folder on exit, set `ts.global.delete.folder.on.exit` to `true`.
```
I'm self-taught (in English) so I have no idea what is right, but `the` feels wrong there, however you decide!

Reply:
Rephrased, see previous thread.